index.bs

```diff
@@ -682,30 +682,25 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
-1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=]
-    currently available on this platform.
+1. Let |lifetimeTimer| be a timer for |adjustedTimeout| milliseconds.
 
-1. Let |selectedAuthenticators| be a new [=ordered set=].
+1. Let |issuedRequests| be a new [=ordered set=].
+
+1. If <code>|options|.{{MakePublicKeyCredentialOptions/authenticatorSelection}}</code> is
+    [=present|present=], [=set/for each=] |authenticator| that becomes available on this
+    platform during the lifetime of |lifetimeTimer| and do the following:
 
-1. If |currentlyAvailableAuthenticators| [=list/is empty=], return a {{DOMException}} whose name is
-    "{{NotFoundError}}", and terminate this algorithm.
+    Issue: The definitions of "lifetime of" and "becomes available" are intended to represent how
+    devices are hotplugged into browsers, and are under-specified. Resolving this with good definitions
```

Comment:
If we intend to address what hotplug is after CR, I'd recommend giving an example such as NFC which people relate to more.